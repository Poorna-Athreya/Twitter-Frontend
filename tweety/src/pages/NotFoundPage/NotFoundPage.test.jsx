import React from 'react';
import { render } from '@testing-library/react';
import NotFoundPage from './index';

describe('NotFoundPage', () => {
  it('should take a snapshot of NotFoundPage', () => {
    const { asFragment } = render(<NotFoundPage />);

    expect(asFragment(<NotFoundPage />)).toMatchSnapshot();
  });
});
