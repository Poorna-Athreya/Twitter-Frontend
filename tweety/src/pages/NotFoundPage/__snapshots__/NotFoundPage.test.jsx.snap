// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage should take a snapshot of NotFoundPage 1`] = `
<DocumentFragment>
  <div
    class="not-found"
  >
    Error 404, Page not Found!
  </div>
</DocumentFragment>
`;
